[Telink] Fix binary operations for OTA LZMA dual-mode

s07641069 opened the pull request:
**Change overview**

- Move merge of Zigbee binary from CMake into process_binaries.py
- Add partition for Zigbee binary (TELINK_ZB_PARTITION_ADDR) instead of hardcoded values

s07641069 committed:
[Telink] Fix binary operations for OTA LZMA dual-mode

fengtai-telink commented on src/platform/telink/tlsr9528a_4m_flash.overlay:
```diff
@@ -20,6 +15,12 @@
+			label = "image-zb";
+			// Zigbee partition
+			// End address should be same as of slot0 partition
+			reg = <0x154000 0x65000>;
```
0x154000 + 0x65000 = 0x1b9000，this zigbee partition address seems to cover 'trailor and move sector'(20k)

fengtai-telink replied:
@haiwentelink Can you help me confirm it ?

haiwentelink replied:
yes , need to adjust to reg = <0x154000 0x60000>;

fengtai-telink replied:
had update the reg size of zb partition to 0x60000

fengtai-telink committed:
riscv: telink: adjust the logic for firmware merging

 - update the reg size of zb partition to 0x60000 .
 - fix the build error for tlsr9518adk80d .

Signed-off-by: Fengtai Xie <fengtai.xie@telink-semi.com>

fengtai-telink commented:
Hi @damien0x0023, can you help me to review and test(If you have already done it, you can skip it) this PR.  I had modify some code to assess CI, so i need to ensure that the functionality is not affected.

fengtai-telink committed:
riscv: telink: adjust the zigbee nvs partition

 - add zigbee nvs partition for tl3218x .
 - adjust the name of zigbee nvs partition .

Signed-off-by: Fengtai Xie <fengtai.xie@telink-semi.com>
